libsolidity/formal/EncodingContext.cpp

```diff
@@ -55,6 +55,18 @@ unsigned EncodingContext::newUniqueId()
 	return m_nextUniqueId++;
 }
 
+void EncodingContext::clear()
+{
+	m_variables.clear();
+	m_expressions.clear();
+	m_globalContext.clear();
+	m_assertions.clear();
+	m_solver = nullptr;
+	m_accumulateAssertions = true;
+	resetUniqueId();
+	m_state.reset();
+}
+
```

Comment:
Clearly that method was never used, it would be better to remove it entirely instead of adding unused implementation detail IMO.